javascript-timers/interval.js

```diff
@@ -0,0 +1,23 @@
+const textInterval = setInterval(changeTextCountdown, 900);
+
+const $countdownDisplay = document.querySelector('.countdown-display');
+function changeTextCountdown() {
+  switch ($countdownDisplay.textContent) {
+    case '4':
+      $countdownDisplay.textContent = '3';
+      break;
+
+    case '3':
+      $countdownDisplay.textContent = '2';
+      break;
+
+    case '2':
+      $countdownDisplay.textContent = '1';
+      break;
+
+    case '1':
+      $countdownDisplay.textContent = '~Earth Beeeelooowww Us~';
+      clearInterval(textInterval);
+      break;
+  }
+}
```

Comment:
Works, but a bit more complicated than necessary. What if we wanted the count to start at a higher number like 10? Now you would need to back and add switch statements for each of those cases. It'd be better to utilize a `count` variable to keep track and just set it's `textContent` with that.

```
let count = 4;
let countdownId = null;
const $countdownDisplay = document.querySelector('.countdown-display');

function countDown() {
  count--;
  if (count > 0) {
    $countdownDisplay.textContent = count;
  } else {
    $countdownDisplay.textContent = '~Earth Beeeelooowww Us~';
    clearInterval(countdownId);
  }
}

countdownId = setInterval(countDown, 1000);
```